RPG-game-Hung/classes/actions.py

```diff
@@ -0,0 +1,27 @@
+from termcolor import colored
+
+from classes.colors import Color
+
+
+class Action:
+    text = None  # For printing during player's action selection
+    collection = {
+        1: ["Slap", "target.slapped()"],
+        2: ["Magic", "Magic.enchant(attacker,target)"],
+        3: ["Item", "Item.use(attacker,target)"],
+    }
+
+    @staticmethod
+    def show_actions():
```

Comment:
Why did you want to make this `@staticmethod`? This method is interacting with `collection` variable of the class, it should be better to use a regular method here.